breakpoint_twa.hpp

```diff
@@ -83,13 +83,7 @@ class bp_twa {
 
       auto old_n = src_->get_named_prop<std::string>("automaton-name");
       if (old_n)
-      {
-      std::stringstream ss;
-      ss << (cut_det_ ? "cDBA for " : "sDBA for ") << old_n;
-
-      std::string * name = new std::string(ss.str());
-      res_->set_named_prop("automaton-name", name);
-      }
+        *old_n = (cut_det_ ? "cDBA for " : "sDBA for ") + *old_n;
```

Comment:
This patch is wrong because it updates the old automaton instead of the result.  However in practice the original code had no effect already: 
- `old_n` is always empty because main calls `scc_filter()` first
- a similar piece of code in `main()` was attempting to set the name, and failed for the same reason

This is fixed by 391da98388.